pkg/util/pravegacluster_test.go

```diff
@@ -175,6 +175,12 @@ var _ = Describe("pravegacluster", func() {
 		})
 
 	})
+	Context("ReadinessHealthcheckCommand()", func() {
```

Comment:
e2e tests are failing on travis... please fix if needed

Reply:
yes, they were getting timeout on restarting them again it passed.